Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and '$(RUNTIME_PATH)' != ''">%RUNTIME_PATH%\</TestRuntimeEnvVar>
```

Comment:
The trouble I see with your current approach is that RunTests.cmd will have different values depending on were you last built, in VS or command line. I still think I would prefer we just isolate the change to the StartProgram/StartArgument properties as those are only used by VS.